Assets/Code/MinigameTanglesLair/GameBehavior.cs

```diff
@@ -143,17 +161,20 @@ private void Start()
         }
 
         Image numberObject = GameObject.Find("Canvas/LevelNumber/Number").GetComponent<Image>();
-        if (GameData.getCurrLevel() < CTConstants.N_LEVELS) {
+        if (GameData.getCurrLevel() < CTConstants.N_LEVELS) 
+        {
             numberObject.sprite = numberSprites[GameData.getCurrLevel()];
         }
 
         if (GameData.getCurrLevel() > CTConstants.N_LEVELS)
         {
+            // Load the Circuits Menu if current game level > number of levels
             SceneManager.LoadScene("Circuits_Menu");
             return; // sorry david :/
         }
         else if (GameData.getCurrLevel() <= 9)
         {
+            Debug.Log("if statement b");
```

Comment:
While this comment (as well as the ones on lines 222, 237, & 323) can be helpful with debugging something in particular, afterwards it just serves to overflow the log with information that's not relevant most of the time. Make sure to keep logs (at least the ones that you leave in when merging) are descriptive, concise, and informative.